components/num-util/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "icu-num-util"
+description = "Low-level tools for dealing with numbers"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+smallvec = "1.4.1"
+static_assertions = "1.1.0"
+more-asserts = "0.2.1"
+
+[dev-dependencies]
+rand = "0.7.3"
+criterion = "0.3"
+rand_pcg = "0.2.1"
+rand_distr = "0.2.2"
```

Comment:
OK.  On the other hand, at least with runtime dependencies like `smallvec`, maybe it's better to enforce the minor and patch versions, because code size can and will change between `smallvec` minor and patch versions, which can and will create inconsistencies when people run those tests.

I changed the dev dependencies as suggested, but left the runtime dependencies with patch versions.

Follow-up: https://github.com/unicode-org/icu4x/pull/141

Reply:
Changed to major.minor based on conclusion in #175